e2e/rook-test-framework/utils/exec_utils.go

```diff
@@ -0,0 +1,212 @@
+package utils
+
+import (
+	"bufio"
+	"bytes"
+	"fmt"
+	"os/exec"
+	"strings"
+	"syscall"
+	"log"
+	"github.com/rook/rook/e2e/rook-test-framework/objects"
+)
+
+func ExecuteCommand(cmdStruct objects.Command_Args) objects.Command_Out {
+	var outBuffer, errBuffer bytes.Buffer
+
+	cmd := exec.Command(cmdStruct.Command, cmdStruct.CmdArgs...)
+
+	cmd.Env = append(cmd.Env, cmdStruct.EnvironmentVariable...)
+
+	stdOut, err := cmd.StdoutPipe()
+
+	if err != nil {
+		return objects.Command_Out{StdErr: errBuffer.String(), StdOut: outBuffer.String(), Err: err}
+	}
+
+	stdin, err := cmd.StdinPipe()
+
+	if err != nil {
+		return objects.Command_Out{Err: err}
+	}
+
+	defer stdOut.Close()
+
+	scanner := bufio.NewScanner(stdOut)
+	go func() {
+		for scanner.Scan() {
+			outBuffer.WriteString(scanner.Text())
+			fmt.Printf("%s\n", scanner.Text())
+		}
+	}()
+
+	stdErr, err := cmd.StderrPipe()
+
+	if err != nil {
+		return objects.Command_Out{StdErr: errBuffer.String(), StdOut: outBuffer.String(), Err: err}
+	}
+
+	defer stdErr.Close()
+
+	stdErrScanner := bufio.NewScanner(stdErr)
+	go func() {
+		for stdErrScanner.Scan() {
+
+			txt := stdErrScanner.Text()
+
+			if !strings.Contains(txt, "no buildable Go source files in") {
```

Comment:
that's an odd message to be looking for in a general ExecuteCommand utils function...comment about why it's needed?